react-backend/database/queries/users_chart_all.js

```diff
@@ -13,7 +13,7 @@ const getYearOfBirth0_17 = today.getFullYear() - 17;
 const getYearOfBirth18_34 = today.getFullYear() - 34;
 const getYearOfBirth35_50 = today.getFullYear() - 50;
 const getYearOfBirth51_69 = today.getFullYear() - 69;
-const getYearOfBirth70_more = today.getFullYear() - 70;
+// const getYearOfBirth70_more = today.getFullYear() - 70;
```

Comment:
Why's this been commented out?

Reply:
It was an unused variable (linting error), but it seems like something that got left around after an implementation change, and it may in future hint at something 🤷‍♂️